src/legendCreation.js

```diff
@@ -1,4 +1,4 @@
-$(window).one('load', function() {
+window.onload = function() {
```

Comment:
Haven't tested it out yet. Should I revert this back if it doesn't throw any errors?